src/SprykerEco/Zed/Computop/Business/Payment/Reader/ComputopPaymentReader.php

```diff
@@ -50,9 +50,10 @@ public function isComputopPaymentExist(QuoteTransfer $quoteTransfer): QuoteTrans
      */
     protected function getPaymentTransfer(QuoteTransfer $quoteTransfer): TransferInterface
     {
-        $paymentSelection = $quoteTransfer->getPayment()->getPaymentSelection();
+        $paymentTransfer = $quoteTransfer->getPaymentOrFail();
```

Comment:
don't we need to support item-level payments?

Reply:
Won't change as discussed